app/src/main/java/org/thunderdog/challegram/util/text/Text.java

```diff
@@ -2104,8 +2143,8 @@ public void locatePart (Rect outRect, TextPart part, int compareMode) {
         int bound = getBackgroundPadding(defaultTextColorSet, entity, compareMode != TextEntity.COMPARE_MODE_NORMAL, false);
         outRect.top -= bound;
         outRect.bottom += bound;
-        outRect.left = parts.get(start).getX();
-        outRect.right = parts.get(end).getX() + (int) parts.get(end).getWidth();
+        // outRect.left = parts.get(start).getX();
+        // outRect.right = parts.get(end).getX() + (int) parts.get(end).getWidth();
```

Comment:
This fix introduces the incorrect behavior: when there are multiple `TextPart` on the same line that generate the same background highlight, the wrong coordinates might be pushed into `outRect` if the one in the middle is pressed.

Reply:
Resolved in d3fe351